runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -527,15 +545,21 @@ private void updateConfig()
 
 			this.configCallers = config.callers();
 			relationColorHashMap.put(PlayerRelation.CALLER, config.callerColor());
-			locationHashMap.put(PlayerRelation.CALLER, config.callerHighlightOptions().toArray());
+			if (config.callerHighlightOptions() != null)
+			{
+				locationHashMap.put(PlayerRelation.CALLER, config.callerHighlightOptions().toArray());
+			}
```

Comment:
if (this.callerHighlightOptions() != null)
			{
				locationHashMap.put(PlayerRelation.CALLER, this.callerHighlightOptions.toArray());
			}